### PR TITLE
AA-361: shift dates banner tests

### PR DESCRIPTION
**Pull request by ciduarte**

### CODE REVIEW

File: src/course-home/dates-tab/DatesTab.test.jsx [hunk 1]
```diff
@@ -128,4 +127,77 @@ describe('DatesTab', () => {
...
+
+    it('renders resetDatesBanner', async () => {
+      datesTabData.datesBannerInfo = {
+        contentTypeGatingEnabled: true,
+        missedDeadlines: true,
+        missedGatedContent: false,
+        verifiedUpgradeLink: 'http://localhost:18130/basket/add/?sku=8CF08E5',
+      };
+
+      axiosMock.onGet(`${getConfig().LMS_BASE_URL}/api/course_home/v1/dates/${courseId}`).reply(200, datesTabData);
```
Comment (mikix) [inline src/course-home/dates-tab/DatesTab.test.jsx:195]: How easy is it to test the failure case, with toast and all that?

**Review by mikix — APPROVED** [review]
Yay tests!

File: src/course-home/dates-tab/DatesTab.test.jsx (hunk 1 above)
Comment (ciduarte) [reply src/course-home/dates-tab/DatesTab.test.jsx:195]: Ooh if we actually click the button? Doesn't seem too bad at first glance I'll attempt it

Comment (mikix) [reply src/course-home/dates-tab/DatesTab.test.jsx:195]: Oh right - yeah. The line I commented on wasn't what I meant to comment on. I was just seeing all these 200s and thought "what if resetting failed" - didn't realize this line was just normal dates info.

Comment (ciduarte) [reply src/course-home/dates-tab/DatesTab.test.jsx:195]: Ooooh I see. I added in a test about clicking the shift dates button anyways (it feels a little redundant but probably fine to leave it)

**Pull request merged**